lib/chef/provider/aws_key_pair.rb

```diff
@@ -1,6 +1,5 @@
 require "chef/provider/lwrp_base"
 require "chef/provisioning/aws_driver/aws_provider"
-require "aws-sdk"
```

Comment:
This should be require "aws-sdk-ec2"

Reply:
At the time I actually removed this deliberatly, since the class doesn't _actually_ use anything from aws-sdk-* directly at all, but rather the things provided by aws_provider.  Still, not real harm in specifying it.